crates/router/src/connector/payme/transformers.rs

```diff
@@ -907,15 +915,17 @@ impl<F, T>
 #[derive(Debug, Serialize)]
 pub struct PaymentCaptureRequest {
     payme_sale_id: String,
-    sale_price: i64,
+    sale_price: MinorUnit,
 }
 
 impl TryFrom<&PaymeRouterData<&types::PaymentsCaptureRouterData>> for PaymentCaptureRequest {
     type Error = error_stack::Report<errors::ConnectorError>;
     fn try_from(
         item: &PaymeRouterData<&types::PaymentsCaptureRouterData>,
     ) -> Result<Self, Self::Error> {
-        if item.router_data.request.amount_to_capture != item.router_data.request.payment_amount {
+        if item.router_data.request.minor_amount_to_capture
+            != item.router_data.request.minor_payment_amount
```

Comment:
Can we do something to remove the `direct` payment amount context here?

Reply:
since, there is no conversion here we can let it stay i suppose! evaluation of the values can be a part of transformers.